stages/test/test_kickstart.py

```diff
@@ -30,3 +31,41 @@ def test_kickstart(tmp_path, test_input, expected):
     with open(os.path.join(tmp_path, ks_path), encoding="utf-8") as fp:
         ks_content = fp.read()
     assert ks_content == expected + "\n"
+
+
+@pytest.mark.parametrize("test_data,expected_err", [
+    # BAD pattern, ensure some obvious ways to write arbitrary
+    # kickstart files will not work
+    ({"clearpart": {"disklabel": r"\n%pre\necho p0wnd"}}, r"p0wnd' does not match"),
```

Comment:
It seems right now we allow strings with \n in some of the kickstart options. So unless I'm missing something and the \n are filtered elsewhere exiting options may also be restricted a bit more (please tell me :)

Reply:
Every command should be on a single line. `\n` would only be valid when writing script sections.